### PR TITLE
Feature/set relationship on term

### PR DESCRIPTION
**Pull request by straleyb**

### CODE REVIEW

File: app/views/relationships/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<% content_for :title do %>
+  Relationships
+<% end %>
+<h1>Relationships</h1>
+<%- if current_user && current_user.admin?  %>
+  <%= link_to "Create Relationship", new_relationship_path, {:class=>'btn btn-default'} %>
```
Comment (revgum) [inline app/views/relationships/index.html.erb]: Remember to eliminate this and replace it with some helpful text directing the user to create a relationship from one of the terms.

File: app/controllers/relationships_controller.rb [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+  delegate :term_form_repository, :to => :term_injector
+  skip_before_filter :require_admin, :only => [:review_update, :mark_reviewed]
+
+  include GitInterface
+  def index
+    #Grab all relationships
+    @relationships = all_relationships_query.call
+  end
+
+  def new_parent_relationship
```
Comment (revgum) [inline app/controllers/relationships_controller.rb]: Just leaving a note related to the discussion of condensing these actions into 1, using a select (or hidden input) to set `hier_parent` or `heir_child` when submitting the form. Eliminate duplicate related views, routes, etc.

File: app/views/vocabularies/new.html.erb [hunk 3]
```diff
@@ -3,4 +3,4 @@
 <% end %>
 <h1> Create New Vocabulary </h1>
 
-<%= render :partial => "form", :locals => { :vocabulary => @vocabulary } %>
+<%= render :partial => "form", :locals => { :relationship => @relationship } %>
```
Comment (wickr) [inline app/views/vocabularies/new.html.erb]: Should adding a New Vocabulary have been changed to the relationship local?

Comment (straleyb) [inline app/views/vocabularies/new.html.erb]: Yeah, I didn't even notice I changed this. I must have thought I was in the relationships new page. This is what broke a ton of tests locally. 👍

**Pull request merged**